sstables/exceptions.hh

```diff
@@ -21,6 +22,9 @@ public:
     malformed_sstable_exception(sstring msg, sstring filename)
         : malformed_sstable_exception{format("{} in sstable {}", msg, filename)}
     {}
+    malformed_sstable_exception(sstring msg, component_name filename)
+        : malformed_sstable_exception{format("{} in sstable {}", msg, filename)}
+    {}
```

Comment:
the commit message reads:

> next patches fill

is "fill" a typo? just wanted to understand what this commit message really means..

Reply:
Should be "will". ~~Fill wix~~ Will fix :)